docs/contribute.rst

```diff
@@ -17,11 +17,13 @@ Install the dev dependencies
     $ pip install -r requirements-dev.txt
 
 Make sure the tests pass.  The ``@failing`` tag is used for tests that are
-supposed to fail.
+supposed to fail. The ``@no-simple`` tag is used for tests that can't run
+with ``--simple`` flag.
 
 .. code:: bash
 
     $ python manage.py behave --tags=~@failing  # skip failing tests
+    $ python manage.py behave --simple --tags=~@failing --tags=~@no-simple
```

Comment:
@mixxorz Without adding to much magic, can and should we make a `--tags=~@no-simple` implicit with `--simple`? I believe that would be intuitive. It may be a bit magical, though.